modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/decisions_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+    module V0
+        class DecisionsController < ApplicationController
+            before_action :set_power_of_attorney_request, only: [:show, :create]
+
+            def create
+                if @power_of_attorney_request.resolution.present?
+                    render json: { error: 'Resolution already exists' }, status: :unprocessable_entity
```

Comment:
In the near future we will have more streamlined exception handling with [`rescue_from`](https://guides.rubyonrails.org/action_controller_advanced_topics.html#rescue-from). You could try this now, but if you want to handroll the error responses very temporarily that's fine too.